docs/installation.rst

```diff
@@ -63,6 +63,10 @@ libspotify in the homebrew/binary tap::
 
     brew install homebrew/binary/libspotify
 
+Install libffi as a dependency::
+
+    brew install libffi
+
```

Comment:
First, libffi is provided by OS X by default. The Homebrew formula for libffi is "keg-only". Do you know for a fact that the libffi bundled with OS X isn't good enough?

Second, instructions about libffi, which is a cffi/pyspotify dep, should not be in the middle of the libspotify section.